### ISSUE TITLE
/gap for Spectators

### ISSUE THREAD

**Issue by SethTheLasersloth** [Makkuusen/TimingSystem#126]
Idea I had, if there was like a "/gap <driver 1> <driver 2>" command or something, which would display the difference in time between the two drivers, in a Race (and also have it function similarly for Quali too). 
For example, the output into chat for Races can be something like "Makkuusen is 24.6s ahead of The_Lasersloth".
A Quali /gap command output example can be "Makkuusen is 0.35s ahead of The_Lasersloth".

It could be a useful command for spectators, being able to quickly calculate the gaps, _especially_ useful for keeping track of drivers with different numbers of pits.

**Comment by Makkuusen** [comment]
Yeah, I think #48 might partially solve the issue presented, but a command to se gap might also be a good idea to have.

**Comment by JustBru00** [comment]
This could be implemented pretty easily using TimingSystemAPI.getDriverDetailsFromHeat().
